spec/fixtures/controllers/applications/lease_up_application_domain.json

```diff
@@ -4,150 +4,196 @@
   "status": "Submitted",
   "preferences": [
     {
-      "zip_code": null,
-      "state": null,
+      "application_member_id": null,
```

Comment:
@akegan Content changed as well. sorry I realize that makes this impossible to review. I think I alphabetized to be able to diff them with firefox console logs but I wasn't thinking about what it would do to the diff.

I can put up a temporary PR that just shows the diff between alphabetized master json <-> these changes? That will only take a few minutes, let me know if that's useful.

Reply:
it's OK, thanks for the offer though!